common/models/user.js

```diff
@@ -557,6 +571,9 @@ module.exports = function(User) {
       cb(err);
       return cb.promise;
     }
+    if (options.password && options.password.length > MAX_PASSWORD_LENGTH) {
+      this.validatePassword(options.password);
```

Comment:
This is exactly what I'm talking about to prevent what weird try/catch on the User.create.